### PR TITLE
provision: add debug option for virt-install

### PR DESCRIPTION
**Pull request by zhlhahaha**
cc: @brianmcarey @dhiller

### CODE REVIEW

**Comment by zhlhahaha** [comment]
Hi @brianmcarey, I have updated the PR. I also modify the jobs in presubmit and postsubmit according to the changes in https://github.com/kubevirt/project-infra/pull/2760

File: cluster-provision/images/vm-image-builder/customize-image.sh [hunk 1]
```diff
@@ -67,6 +71,10 @@ virt-install \
   ${buildconfig} \
   ${consoleconfig}
 
+if [[ ${DEBUG} = "yes" ]]; then
```
Comment (brianmcarey) [inline cluster-provision/images/vm-image-builder/customize-image.sh:74]: needs to be updated to `true`

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Comment by zhlhahaha** [comment]
@brianmcarey do we need to ping any other to get this PR merge?

**Review by brianmcarey — APPROVED** [review]
/approve

**Comment by zhlhahaha** [comment]
Thanks @brianmcarey

**Pull request merged**